common/logisticspipes/items/ItemGuideBook.java

```diff
@@ -0,0 +1,25 @@
+package logisticspipes.items;
+
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.EnumActionResult;
+import net.minecraft.util.EnumHand;
+import net.minecraft.world.World;
+
+import logisticspipes.LogisticsPipes;
+import logisticspipes.network.GuiIDs;
+
+public class ItemGuideBook extends LogisticsItem{
+	public ItemGuideBook(){
+		this.setMaxStackSize(1);
+	}
+
+	@Override
+	public ActionResult<ItemStack> onItemRightClick(World world, EntityPlayer player, EnumHand hand) {
+		ItemStack itemstack = player.getHeldItem(hand);
+		if(!world.isRemote){return new ActionResult<ItemStack>(EnumActionResult.SUCCESS, itemstack);}
```

Comment:
`new ActionResult<ItemStack>` → `new ActionResult<>`

Reply:
Hahaha IDEA has the same opinion
![image](https://user-images.githubusercontent.com/6233500/59119121-e5a62680-8949-11e9-8caf-2306ebdb7b72.png)

Fixed